ledger/participant-integration-api/src/main/resources/db/migration/h2database-appendonly/V1__Append_only_schema.sql

```diff
@@ -539,3 +541,8 @@ SELECT
     exercise_result_compression
 FROM participant_events_non_consuming_exercise
 ;
+
+CREATE TABLE string_interning (
```

Comment:
Wouldn't `identifier_interning` be a better name for this purpose? `string_interning` sounds quite generic.

Reply:
I think string interning is ok, it is a generic feature, and it is exclusively about string identifiers. (not to mention changing this name would need to ripple through many places now)